src/CohortList.js

```diff
@@ -0,0 +1,120 @@
+import Cohort from './Cohort.js'
+
+class CohortList {
+  constructor() {
+    this.list = []
+    this.studentsList = []
+  }
+
+  // Create Cohort
+  createCohort(name) {
+    if (name.length === 0) {
+      throw new Error('Enter name for create new cohort')
+    }
+
+    if (this.list.find((item) => item.name === name)) {
+      throw new Error('Cohort with this name has already exist')
+    }
+
+    const cohort = new Cohort(this.list.length + 1, name)
+
+    this.list.push(cohort)
+
+    return cohort
+  }
+
+  // Find Cohort By Name
+  getCohortByName(name) {
+    const findCohort = this.list.find((item) => item.name === name) || null
+
+    if (!findCohort) {
+      throw new Error('Cohort not Found')
+    }
+
+    return findCohort
+  }
+
+  // Add student
+  addStudent(student, cohortName) {
+    if (
+      this.studentsList.find((item) => item.studentId === student.studentId)
+    ) {
+      throw new Error('This student has already added to cohort')
+    }
+
+    const findCohort = this.getCohortByName(cohortName)
+
+    if (findCohort.studentsList.length === 24) {
```

Comment:
24 is a magic number here, we don't know what 24 means. I would move this to a property named `studentCapacity`

Reply:
Do you mean just creat new variable studentsCapacity like that:

```
const studentCapacity = 24
```

Reply:
You could, but I think it would be better as a class member so you can define a cohort's capacity when instantiating it. Stick it in the constructor